2019-Spring-WashU.md

```diff
@@ -337,7 +337,7 @@ Most class periods will be split between lecture/discussion about Open Source So
     * Topic: Challenges for OSS maintainers - real world examples
   * Presenter 2: David Huang
     * Topic: Be careful who you trust (npm event-stream)
-  * Presenter 3:
+  * Presenter 3: 
```

Comment:
Oops: errant space. Can you please remove this so your PR doesn't change this line?

Reply:
M'eh... No one else has tried to grab that spot, so merging...